src/main/scala/com/fortysevendeg/scalacheck/datetime/instances/j8.scala

```diff
@@ -0,0 +1,15 @@
+package com.fortysevendeg.scalacheck.datetime.instances
+
+import com.fortysevendeg.scalacheck.datetime.typeclasses._
+import java.time._
+import java.time.temporal.ChronoUnit.MILLIS
+
+trait J8Instances {
+
+  implicit val j8ForDuration: ScalaCheckDateTimeInfra[ZonedDateTime, Duration] = new ScalaCheckDateTimeInfra[ZonedDateTime, Duration] {
```

Comment:
I've had issues in the past with type inference when doing things that way (admittedly with more complex types than this). I seem to recall that `implicit val...` is the suggested way to do it, though I can't find any evidence to back that up. Doing it my way allows explicit setting of the desired type, which I like, so I'll probably stick with that for now.